src/content/certifications/exam-faqs/terraform-associate-004.mdx

```diff
@@ -0,0 +1,132 @@
+| **Expiration**      | 2 years                                                                           |
+| **Release Date**    | January 8,2026                                                                    |
+
+## Exam Objectives
+
+|  |  |
+|-----------|-------------|
+| **1** | **Infrastructure as Code (IaC) with Terraform** |
+| 1a | Explain what IaC is |
+| 1b | Describe the advantages of IaC patterns |
+| 1c | Explain how Terraform manages multi-cloud, hybrid cloud, and service-agnostic workflows |
+| **2** | **Terraform fundamentals** |
+| 2a | Install and version Terraform providers |
+| 2b | Describe how Terraform uses providers |
+| 2c | Write Terraform configuration using multiple providers |
+| 2d | Explain how Terraform uses and manages state |
+| **3** | **Core Terraform workflow** |
+| 3a | Describe the Terraform workflow |
+| 3b | Initialize a Terraform working directory |
+| 3c | Validate a Terraform configuration |
+| 3d | Generate and review an execution plan for Terraform |
+| 3e | Apply changes to infrastructure with Terraform |
+| 3f | Destroy Terraform-managed infrastructure |
+| 3g | Apply formatting and style adjustments to a configuration |
+| **4** | **Terraform configuration** |
+| 4a | Use and differentiate `resource` and `data` blocks |
+| 4b | Refer to resource attributes and create cross-resource references |
+| 4c | Use variables and outputs |
+| 4d | Understand and use complex types |
+| 4e | Write dynamic configuration using expressions and functions |
+| 4f | Define resource dependencies in configuration |
+| 4g | Validate configuration using custom conditions |
+| 4h | Understand best practices for managing sensitive data, including secrets management with Vault |
+| **5** | **Terraform modules** |
+| 5a | Explain how Terraform sources modules |
+| 5b | Describe variable scope within modules |
+| 5c | Use modules in configuration |
+| 5d | Manage module versions |
+| **6** | **Terraform state management** |
+| 6a | Describe the local backend |
+| 6b | Describe state locking |
+| 6c | Configure remote state using the backend block |
+| 6d | Manage resource drift and Terraform state |
+| **7** | **Maintain infrastructure with Terraform** |
+| 7a | Import existing infrastructure into your Terraform workspace |
+| 7b | Use the CLI to inspect state |
+| 7c | Describe when and how to use verbose logging |
+| **8** | **HCP Terraform** |
+| 8a | Use HCP Terraform to create infrastructure |
+| 8b | Describe HCP Terraform collaboration and governance features |
+| 8c | Describe how to organize and use HCP Terraform workspaces and projects |
+| 8d | Configure and use HCP Terraform integration |          |
+
+## Requirements for Attending an Exam
+
+Review the [rules and policies](https://hashicorp-certifications.zendesk.com/hc/en-us/articles/26234761626125-Exam-appointment-rules-and-requirements) for taking HashiCorp certification exams.
+
+## Renewing Your Certification
```

Comment:
The recertification section for **004** is weird at this point. It tells about scenarios that don't yet exist and doesn't tell about scenarios that currently exist. I am wondering if it is best just to remove the entire section for now and either reinstate or rework/reinstate it later once it has applicable scenarios.